tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -985,6 +985,30 @@ everflow/test_everflow_testbed.py::TestEverflowV4EgressAclEgressMirror::test_eve
     conditions:
       - "https://github.com/sonic-net/sonic-mgmt/issues/17034 and 't2' not in topo_name"
 
+everflow/test_everflow_testbed.py::TestEverflowV4EgressAclEgressMirror::test_everflow_fwd_recircle_port_queue_check:
+    skip:
+      reason: "Test not supported on non broadcom-dnx and non voq platforms"
+      conditions_logical_operator: "OR"
+      conditions:
+        - "(asic_subtype not in ['broadcom-dnx'])"
+        - "('voq' not in switch_type)"
```

Comment:
Is this test expected to run on single-asic disagg-t2 boxes?
I'm wondering because they will be both `voq` and `broadcom-dnx` but will there still be an `Ethernet-Rec` port on those boxes?